ansible/roles/postgresql/defaults/main.yml

```diff
@@ -9,3 +9,6 @@ postgresql_pid_file: "{{ postgresql_dir_path }}/{{ postgresql_version }}-main.pi
 postgresql_dbs:
   - "{{ localsettings.PG_DATABASE_NAME }}"
   - "{{ localsettings.REPORT_DATABASE_NAME }}"
+pg_bouncer_config_home: /etc/pgbouncer
+pg_bouncer_ini: "{{ pg_bouncer_config_home }}"/pgbouncer.ini
```

Comment:
This line breaks ansible deploys FYI. I think it should be `"{{ pg_bouncer_config_home }}/pgbouncer.ini"` with the quotes around the whole thing. (I was able to get around it by winding back and cherry picking the thing i wanted to deploy.)